docker-compose.yml

```diff
@@ -141,10 +152,27 @@ services:
         AwEHoUQDQgAEdakNgjerNLFXy1uMfwLjPd5/pBTbIEoZMdMV/o6s+uoMg776LF+v
         gtbfIaZbhDcJElzHwpnqGfHYU+cM4KyurA==
         -----END EC PRIVATE KEY-----
+    restart: on-failure
     depends_on:
+      lando-init:
+        condition: service_completed_successfully
       lando.db:
         condition: service_healthy
 
+  lando-init:
+    user: "${UID}:${GID}"
+    image: mozilla/lando:latest  # TODO: this does not exist yet.
+    command: lando setup_dev
+    env_file:
+      - ../lando/.env
+    environment:
+      <<: *base-lando-vars
+    depends_on:
+      lando.db:
+        condition: service_healthy
+      phabricator.test:
+        condition: service_healthy
+
```

Comment:
As discussed, might be better to include the `setup_dev` command in the main lando container instead, along with the other development specific commands. Also, this could be combined in the main compose file.